packages/ember-test-app/app/templates/marketing.hbs

```diff
@@ -198,5 +198,51 @@
         </div>
       </div>
     </form>
+
+    <Mktg::ServicePricing
+      @serviceName="Fiber"
+      @serviceType="The Gig"
+      @description="$65/mo"
+      @icon="bi-wifi"
+      as |Addon|
+    >
+      <Addon @addon="Smart Gig" @price="$15/mo" />
+    </Mktg::ServicePricing>
+    <Mktg::ServicePricing
+      @serviceName="TV"
+      @serviceType="Silver"
+      @description="$107/mo"
+      @icon="bi-tv"
+      as |Addon|
+    >
+      <Addon @addon="FireStick" @price="$80" @quantity="2" />
+      <Addon @addon="HBO" @price="$5.99/mo" />
+      <Addon @addon="STARZ" @price="$4.99/mo" />
+      <Addon @addon="Spanish Language Channels" @price="$4.99/mo" />
+    </Mktg::ServicePricing>
+    <Mktg::ServicePricing
+      @serviceName="Phone"
+      @description="$129/mo"
+      @icon="bi-telephone"
+    />
+    <Mktg::ServicePricing
+      @serviceName="Fiber"
+      @serviceType="The Gig"
+      @description="Required"
+      @icon="bi-wifi"
+      @active="true"
+    />
+    <Mktg::ServicePricing
+      @serviceName="TV"
+      @description="Add (optional)"
+      @icon="bi-tv"
+      @descriptionDisabled = {{true}}
+    />
+    <Mktg::ServicePricing
+      @serviceName="Phone"
+      @description="Add (optional)"
+      @icon="bi-telephone"
+      @descriptionDisabled = {{true}}
```

Comment:
I think we collectively decided to use the @active param for determining the "Select", if not active and no selection made, then gray and underlined, if not active and selection is made, bolded price